src/UpdateProcessor/RemoveSticker.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Hobot\UpdateProcessor;
+
+use Hobot\UpdateProcessorInterface;
+use Telegram\Bot\Api;
+use Telegram\Bot\Objects\Update;
+
+class RemoveSticker implements UpdateProcessorInterface
+{   
+    protected $chat_id = array('id1','id2');
```

Comment:
Chat ids should be set through the constructor. Also i suggest using the short array syntax.